SPECS/azurelinux-rpm-macros/gen-ld-script.sh

```diff
@@ -6,26 +6,30 @@
 # gen-ld-script.sh
 # Generate linker script to embed ELF binaries with build metadata
 
-# /usr/lib/rpm/azl/gen-ld-script.sh %{name} %{version} %{_topdir} %{distro_release_version}
+# /usr/lib/rpm/azl/gen-ld-script.sh %{name} %{version}-%{release} %{_topdir} %{distro_release_version}
```

Comment:
is epoch missing?

Reply:
Yes, I'll add the epoch if it is defined for each package